app/services/hyrax/curation_concern.rb

```diff
@@ -17,7 +17,7 @@ class CurationConcern
     # is used.  Once it is used, it becomes immutable.
     # @return [ActionDispatch::MiddlewareStack]
     def self.actor_factory
-      Hyrax::DefaultMiddlewareStack.build_stack
+      @middleware_stack ||= Hyrax::DefaultMiddlewareStack.build_stack
```

Comment:
It's a bit surprising that the ivar is not the same name as the method name. (Not a blocker, just a surprise).

Reply:
I have no issue with renaming this. I think I just went with `middleware_stack` because it's an instance of `ActionDispatch::MiddlewareStack`. I didn't know it was a convention to mirror the method name.

Reply:
It's not quite a convention, but something I personally would've expected (not that I would want to directly access the instance variable). At times, when debugging, I'll do `object.instance_variables` to see what all is there. And method names equalling instance variables is a "nice to have".